demo_grid/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<module>
+    <name>demo_grid</name>
+    <displayName><![CDATA[Demo Grid]]></displayName>
+    <version><![CDATA[1.0.0]]></version>
+    <description><![CDATA[Demonstration of Grid in PrestaShop 1.7.7]]></description>
+    <author><![CDATA[PrestaShop]]></author>
+    <tab><![CDATA[]]></tab>
+    <is_configurable>1</is_configurable>
+    <need_instance>1</need_instance>
```

Comment:
I am not sure that this file is useful. In any case this module don't need instance.